presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java

```diff
@@ -194,7 +194,7 @@ public void preparedInsertApi()
                     param(VARCHAR, "ala ma kot"),
                     param(CHAR, "    ala ma"),
                     param(BOOLEAN, Boolean.TRUE),
-                    param(VARBINARY, null));
+                    param(VARBINARY, "a290IGJpbmFybnk=".getBytes()));
```

Comment:
it would be nice to have something more friendly than `a290IGJpbmFybnk=`, maybe `ala ma kota`? Because of that it is difficult to recognized if output is not a garbage.

Reply:
Also, It is purposeful that this value is valid base64 encoded text? This could be misleading. Better to use something that is more obvious and also less textual, to cover any potential encoding problems. Like `new byte[] { 0, 1, 2, 3, 0, 42, -7 }`

Reply:
@ilfrin  please update this